src/components/calls/services/callsSrv.js

```diff
@@ -117,15 +143,15 @@
                 });
             };
 
-            this.declineCall = function(callsData, hangWebCall) {
+            this.declineCall = function(callsData) {
```

Comment:
the function name is declineCall why the parameter name is callsData and not callData ?